lib/commands.coffee

```diff
@@ -6,6 +6,7 @@ class Commands
     @_subscriptions.add atom.commands.add 'atom-workspace',
       'linter:next-error': @nextError.bind(@)
       'linter:toggle': @toggleLinter.bind(@)
+      'linter:set-bubble-transparent': => @setBubleTransparent()
```

Comment:
You're correct while still missing the point. This solution is valid here, where the other solution is valid always.

"always" being, as long as you're using CoffeeScript classes, and subclasses are created through extension and not some bizarre property copying. I could make up a stupid way to break this, but I've never seen real code that would.